Bugsnag/Payload/BugsnagSession+Private.h

```diff
@@ -47,18 +32,20 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (readwrite, nonnull, nonatomic) BugsnagUser *user;
 
-#pragma mark Methods
+@end
 
-- (void)resume;
+#pragma mark Serialization
 
-- (void)stop;
+/// Produces a session dictionary that contains all the information to fully recreate it via BSGSessionFromDictionary().
+NSDictionary * BSGSessionToDictionary(BugsnagSession *session);
 
-// Representation used in report payloads.
-- (NSDictionary *)toJson;
+/// Parses a session dictionary produced by BSGSessionToDictionary() or added to a KSCrashReport by BSSerializeDataCrashHandler().
+BugsnagSession *_Nullable BSGSessionFromDictionary(NSDictionary *_Nullable json);
 
-/// Full representation of a session suitable for creating an identical session using -initWithDictionary:
-- (NSDictionary *)toDictionary;
```

Comment:
`toDictionary` is used by `bugsnag-unity`

https://github.com/bugsnag/bugsnag-unity/blob/bdd771d30f635f5ead5225186a5196355b888c63/src/BugsnagUnity.m#L369

Reply:
https://github.com/bugsnag/bugsnag-unity/pull/547